src/lib/kclHelpers.ts

```diff
@@ -143,3 +145,15 @@ export async function stringToKclExpression(
 export function getStringValue(code: string, range: SourceRange): string {
   return code.slice(range[0], range[1]).replaceAll(`'`, ``).replaceAll(`"`, ``)
 }
+
+export async function setExperimentalFeatures(
```

Comment:
Little wrapper around the wasm call that handles code exec. Would be nice to unit test this, for now it's covered under the one playwright test.